core/securitylog/package_test.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package securitylog_test
+
+import (
+	"testing"
+
+	gc "gopkg.in/check.v1"
+)
+
+func TestAll(t *testing.T) {
+	gc.TestingT(t)
+}
```

Comment:
Just delete this file, it uses `gc`, but all the tests are written against standard `testing`.